java/src/jmri/Manager.java

```diff
@@ -320,7 +326,9 @@ static public int getSystemPrefixLength(@Nonnull String inputName) throws NamedB
             throw new NamedBean.BadSystemNameException();
         }
         if (!Character.isLetter(inputName.charAt(0))) {
-            throw new NamedBean.BadSystemNameException();
+            if (!UUID.matcher(inputName).find()) {
```

Comment:
`getSystemPrefixLength` gets used a lot, and I'd rather not build-in special purpose code for the long term.  (I've already done a lot of work to move the existing special case code to the section just below, which will be migrated away in a production release or two).  If your EntryExit NamedBeans could start with e.g. "IE" or "IN", then the normal parsing would work fine without any changes throughout JMRI code.

Reply:
I will make the changes.

I am leaning towards **IN**.  Should there be a separator between the prefix and the UUID, such as a dash or colon?  The letters in the UUID are lower case.  Is that an issue?

My current plan is to add the prefix during XML **load()** and remove the prefix during XML **store()**.   This would apply to both the Entry/Exit system name and Conditional References (which this PR is forcing to be the system name).

The store() activity may not be required.  I will need to test for backward compatibility issues before making that determination.

I will add WIP to the title for now.

Reply:
> I am leaning towards IN. Should there be a separator between the prefix and the UUID, such as a dash or colon? The letters in the UUID are lower case. Is that an issue?

Use dash, colon, nothing or any other printable ASCII character as you prefer; I recommend nothing or a dash. The UUID being lower or upper case is not an issue. It really is freeform after the system letter + number (if multiple connections with same system letter) + "N".

Reply:
Thanks!  Let me know if I can help.

Many existing system name patterns (see the [Names page](http://jmri.org/help/en/html/doc/Technical/Names.shtml)) don't use a separator:  LT1, etc.  Some of the ones that are automatically created use a $ or : as a separator: Signal masts create things like IF$shsm:basic:one-searchlight:IH1, automatically created Logix look like IX:AUTO:0001,  etc.  You could use either of those if you'd like, but you don't have to if there's no ambiguity between "system names automatically created when something happens" and "system names that the user manually entered".

Either lower case vs upper case is fine.  The name is case-specific.